### ISSUE TITLE
Incompatible with Storybook v4?

### ISSUE THREAD

**Issue by stephanschubert** [vertexbz/storybook-addon-react-live-edit#1]
I'm getting the following error:

```
ERROR in ./node_modules/storybook-addon-react-live-edit/dist/component/LivePreview.js
Module not found: Error: Can't resolve '@storybook/react/dist/client/preview/error_display'
```

**Comment by mads-thines** [comment]
I'm experiencing the same problem.

**Comment by spatil32** [comment]
Getting the same error.

**Comment by sethwebster** [comment]
Same here.

**Comment by vertexbz** [comment]
Hi, it should be fixed in 2.0.1, thanks for reporting and sorry for delay.
